webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -23,6 +23,17 @@
 from webviz_ert import assets
 
 
+def sort_dataframe(
+    dataframe: pd.core.frame.DataFrame,
+    index: Optional[pd.core.indexes.base.Index],
+    sort_by_column: str,
+) -> Tuple[pd.core.frame.DataFrame, pd.core.indexes.base.Index]:
+    if index is None:
+        index = dataframe[sort_by_column].abs().sort_values().index.copy()
+    dataframe = dataframe.reindex(index)
+    return dataframe, index
```

Comment:
Regarding vertical order, we expect the most important concepts to come first, so the natural order would be to put it after `response_correlation_controller`.

Reply:
good to know, I'll move it down! is that a python style convention, scout convention, equinor convention, or just basic understanding? :innocent: 
my personal intuition is to put helper / utility functions like this one above the "main" or important functions... but that's just me ^^

Reply:
moved it down!